### PR TITLE
Updating the link to download

### PR DESCRIPTION
**Pull request by danbaker7**
changed the download to a Cloud Storage solution like the pricing tool.

### CODE REVIEW

File: articles/cloud-storage/cs-how-install-cifs-ecs.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ CIFS-ECS offers a single caching option:
 
 ## Installing CIFS-ECS
 
-1. Navigate to the following Portal page to download the ZIP file that contains the [CIFS-ECS installer](https://portal.ukcloud.com/support/knowledge_centre/85dc54ff-7d21-43ac-9cfa-88e87de7ba27).
+1. Navigate to the following Portal page to download the ZIP file that contains the [CIFS-ECS installer](https://cas.frn00006.ukcloud.com/Docs/Cloud_Storage/CIFS-ECS.zip?AWSAccessKeyId=438-1048-5-aefff7-1&Expires=1574424921&Signature=sPorRMvkR5S8lLn1D1tmahyzK3g%3D).
```
Comment (suehighmoor) [inline articles/cloud-storage/cs-how-install-cifs-ecs.md]: To be more accurate, I would say:

Navigate to the following URL to download the ZIP file...

**Comment by suehighmoor** [comment]
And have you confirmed that it is OK to provide this ZIP file without any login required? Is it freely accessible to customers elsewhere, or do they need to log in to Dell EMC to get it?

**Comment by danbaker7** [comment]
> And have you confirmed that it is OK to provide this ZIP file without any login required? Is it freely accessible to customers elsewhere, or do they need to log in to Dell EMC to get it?

The download is freely available online, just difficult to find. Which is why I thought having a 'open' link would be okay.

**Comment by suehighmoor** [comment]
Superseded by PR300

**Pull request closed without merging**